js/index.js

```diff
@@ -1 +1,104 @@
 // Your code goes here
+
+//nav enlarges text size on double click, goes back to normal font size when mousing out
+const nav = document.querySelector('nav');
+nav.addEventListener('click', event => {
+    event.target.style.fontSize = "2rem";
+    event.preventDefault();
+})
+
+nav.addEventListener('mouseout', event => {
+    event.target.style.fontSize = "1.6rem";
+})
+
+
+//logo hover effect to blue
+
+
+const logo = document.querySelector('.logo-heading');
+logo.addEventListener('mouseover', event => {
+    event.target.style.color = 'blue';
+})
+
+logo.addEventListener('mouseout', event => {
+    event.target.style.color = 'black';
+    logo.textContent ='Fun Bus';
+})
+
+logo.addEventListener('click', () => {
+    logo.textContent = "Muwhaha";
+});
+
+//adds a border upon imgs loading 
+
+const imgs = document.querySelectorAll('img');
+
+imgs.forEach(img => img.addEventListener('load', event => {
+    event.target.style.border = '2px dashed blue';
```

Comment:
not sure if that's the best usage of the load event.  why not just style the images in the css rather than styling it on load?  just a conceptual question.